src/components/SearchResult/SearchResult.tsx

```diff
@@ -25,8 +24,8 @@ const SearchResult = ({ bib }: SearchResultProps) => {
         paddingBottom: "m",
       }}
     >
-      <CardHeading level="three">
-        <DSLink href={`${PATHS.BIB}/${bib.id}`}>{bib.title}</DSLink>
+      <CardHeading level="h3" size="heading4" url={`${PATHS.BIB}/${bib.id}`}>
+        {bib.title}
```

Comment:
why was this link removed?

Reply:
This is Diego's update in PR #37  but I wasn't pointing my PR to his. The `CardHeading` has a prop for the URL with specific styling so that's why it was moved.